jwst_mast_query/jwst_query.cfg

```diff
@@ -0,0 +1,78 @@
+# specify the instrument. The default set in the code is nircam
+instrument: nircam
+
+# define the output filename
+outrootdir: $JWSTDOWNLOAD_OUTDIR
+outsubdir: 
```

Comment:
Is this just a subdirectory to tag onto the end of outrootdir? Might be worth documenting that.